app/src/main/java/com/kickstarter/viewmodels/ResetPasswordViewModel.kt

```diff
@@ -84,14 +87,20 @@ interface ResetPasswordViewModel {
 
             resetPasswordNotification
                     .compose(errors())
-                    .map { ErrorEnvelope.fromThrowable(it) }
-                    .filter { ObjectUtils.isNotNull(it) }
+                    .map {
+                        // host -> internet related error or otherwise -> invalid email address
+                        if( it !=  null) return@map it.localizedMessage!!.contains("host")
+                        else return@map false
+                    }
+                    //.filter { ObjectUtils.isNotNull(it) }
```

Comment:
nit: remove commented out code

Reply:
Actually bring back the filter, we do want to avoid sending null objects to the `resetError` observable